src/utils/Normalize.scss

```diff
@@ -0,0 +1,8 @@
+html {
+  font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;
```

Comment:
Consider using a more modern font stack that includes system fonts for better performance and user experience. 🖥️